spec/phlex/component_spec.rb

```diff
@@ -387,7 +387,8 @@ def status_emoji
     describe "with custom elements" do
       let :component do
         Class.new Phlex::Component do
-          register_element :trix_editor, :trix_toolbar
+          register_element :trix_editor
+          register_element :trix_toolbar
```

Comment:
Not accepting a splat means we can accept a `tag:` override in case we need the method name to be different from the tag it generates.